vignettes/quitte-data-analysis.Rmd

```diff
@@ -425,9 +430,9 @@ data %>%
     facet_wrap(~ region, scales = 'free_y')
 ```
 
-(Note: You don't have to call the `mip`-functions via the double colon (`::`) 
-operator. This is only needed in this vignette, since `quitte` can't import 
-`mip`.)
+> **Note:** You don't have to call the `mip`-functions via the double colon
+> (`::`) operator.  This is only needed in this vignette, since `quitte` can't
```

Comment:
But do what instead? Load the entire library? I think this could be read as if one already has the mip functions in the environment by default.